examples/advanced/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json

```diff
@@ -24,7 +24,7 @@
   "components": [
     {
       "id": "prostate-learner",
-      "path": "pt.learners.supervised_monai_prostate_learner.SupervisedMonaiProstateLearner",
```

Comment:
Is this fixing a bug?

Reply:
yes, pt no longer works for the current setting